### PR TITLE
Fixed new Parsing Rules RN error

### PR DESCRIPTION
**Pull request by Shellyber**
## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-5775

## Description
Fixed an issue when adding new parsing rules to a pack it was throwing an error

### CODE REVIEW

**Comment by ShahafBenYakir** [comment]
Let's add a unit test

**Comment by Shellyber** [comment]
> Let's add a unit test

@ShahafBenYakir Done

File: demisto_sdk/commands/common/constants.py [hunk 1]
```diff
@@ -644,6 +644,7 @@ class FileType(str, Enum):
 PARSING_RULES_YML_REGEX = (
     rf"{PARSING_RULES_PACKAGE_REGEX}/(?:parsingrule-)?([^/]+)\.yml"
 )
+PARSING_RULES_RULES_REGEX = rf"{PARSING_RULES_PACKAGE_REGEX}\/([^/]+)\.xif"
```
Comment (ShahafBenYakir) [inline demisto_sdk/commands/common/constants.py:647]:
```suggestion
PARSING_RULE_RULES_REGEX = rf"{PARSING_RULES_PACKAGE_REGEX}\/([^/]+)\.xif"
```

Comment (ShahafBenYakir) [reply demisto_sdk/commands/common/constants.py:647]: Worth to also change PARSING_RULES_YML_REGEX to PARSING_RULE_YML_REGEX

**Pull request merged**